src/common/state/object_table.cc

```diff
@@ -86,12 +86,12 @@ void object_info_subscribe(DBHandle *db_handle,
                            RetryInfo *retry,
                            object_info_done_callback done_callback,
                            void *user_context) {
-  ObjectInfoSubscribeData *sub_data = malloc(sizeof(ObjectInfoSubscribeData));
+  ObjectInfoSubscribeData *sub_data = (ObjectInfoSubscribeData *) malloc(sizeof(ObjectInfoSubscribeData));
```

Comment:
same here, might be easier to have the cast and the malloc on the same (second) line.